<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLine</class>
 <widget class="QDialog" name="DialogLine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>238</width>
    <height>219</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a rotation</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>99999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>99999.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>30</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>99999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>99999.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-99999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>99999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-9999.000000000000000</double>
   </property>
   <property name="maximum">
    <double>9999.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>62</width>
     <height>27</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100000000.989999994635582</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Value of b (x, y, z)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>361</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Value of d (x, y, z)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Angle of rotation (in degrees)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogLine</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogLine</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
